src/Inventory/Api/CustomTargetingService.php

```diff
@@ -168,4 +185,24 @@ public function addValuesToKeyById($keyId, $values) {
 
 		return $addedValues;
 	}
+
+	public function removeValueFromKeyById($keyId, $valueId) {
+        $statementBuilder = (new StatementBuilder())->where(
+            'customTargetingKeyId = :customTargetingKeyId AND id = :id'
+        )
+            ->orderBy('id ASC')
+            ->limit(StatementBuilder::SUGGESTED_PAGE_LIMIT)
+            ->withBindVariableValue(
+                'customTargetingKeyId',
+                $keyId
+            )
+            ->withBindVariableValue('id', $valueId);
+
+        $result = $this->customTargetingService->performCustomTargetingValueAction(
+            new \Google\AdsApi\AdManager\v202105\DeleteCustomTargetingValues(),
```

Comment:
I think the code style for this hack-project is that we're adding namespaces at the beginning of the file for classes constructors